include/color_spinor_field.h

```diff
@@ -775,59 +734,34 @@ namespace quda {
        @param[in] b Twisted mass parameter (flavor twist factor, default=0)
        @param[in] c Twisted mass parameter (chiral twist factor, default=0)
       */
-    void packGhost(const int nFace, const QudaParity parity, const int dim, const QudaDirection dir, const int dagger,
-                   qudaStream_t *stream, MemoryLocation location[2 * QUDA_MAX_DIM], MemoryLocation location_label,
+    void packGhost(const int nFace, const QudaParity parity, const int dagger,
+                   qudaStream_t stream, MemoryLocation location[2 * QUDA_MAX_DIM], MemoryLocation location_label,
                    bool spin_project, double a = 0, double b = 0, double c = 0, int shmem = 0);
 
-    void packGhostExtended(const int nFace, const int R[], const QudaParity parity, const int dim,
-                           const QudaDirection dir, const int dagger, qudaStream_t *stream, bool zero_copy = false);
-
     /**
       Initiate the gpu to cpu send of the ghost zone (halo)
       @param ghost_spinor Where to send the ghost zone
-      @param nFace Number of face to send
       @param dim The lattice dimension we are sending
       @param dir The direction (QUDA_BACKWARDS or QUDA_FORWARDS)
-      @param dagger Whether the operator is daggerer or not
       @param stream The array of streams to use
```

Comment:
Update, since given the function declaration change this is no longer an array of streams

Reply:
Good catch (fd5ff4376).